### DISCUSSION TITLE
PiGallery2

### DISCUSSION THREAD

**Discussion by barefootpookie8** [selfhst/icons#105] [New Icon Request]
GitHub: https://github.com/bpatrik/pigallery2
Official: https://bpatrik.github.io/pigallery2/
PNG of Logo: https://github.com/bpatrik/pigallery2/blob/master/docs/assets/icon.png
Inverted PNG of Logo: https://github.com/bpatrik/pigallery2/blob/master/docs/assets/icon_inv.png

**Comment by shollyethan** [comment]
An icon for PiGallery2 already existed, but I went ahead and added an SVG version using the links you provided. Thanks!
